CHANGES.md

```diff
@@ -20,10 +20,11 @@ RPC:
   - serialization/deserialization fixes
   - Updated/fixed integration tests
 
-CI: ([#120](https://github.com/informalsystems/tendermint-rs/issues/120))
-- Moved to GitHub Actions
-- Updated crates.io badges
-- Enabled integration tests in CI with Tendermint-Go node service
+CI:
+- Moved to GitHub Actions ([#120](https://github.com/informalsystems/tendermint-rs/issues/120))
+- Updated crates.io badges ([#120](https://github.com/informalsystems/tendermint-rs/issues/120))
+- Enabled integration tests in CI with Tendermint-Go node service ([#120](https://github.com/informalsystems/tendermint-rs/issues/120))
+- Removed docs folder from CI execution ([#309](https://github.com/informalsystems/tendermint-rs/issues/309))
```

Comment:
General remark: Do we really want to mention every CI improvement in the Changelog? It does not affect anyone user of this repo.

Reply:
LOL, we don't need to, I'm just a busybody. I actually never know what we should mention about CI, because it doesn't affect anyone outside the developers of the repository.

When we cut a release, we can clean this up. I think it's worth keeping them in pending, so developers are aware.

Reply:
I also felt weird adding CI improvements to the changelog, but did it to follow existing precedent. 

But Greg's certainly right that devs should be kept abreast of developments in our CI. Maybe it would make sense to have a separate changelog for the devenv (inclusive of all CI/CD)  -- since it's a piece of software in its own right, whose users are any and all contributors.

Reply:
(Oh man -- sorry for the super tardy reply here... I am cleaning out my inbox and didn't check the context/dates before replying :facepalm: )